hnn_core/extracellular.py

```diff
@@ -28,6 +28,36 @@
 from .externals.mne import _validate_type, _check_option
 
 
+def calculate_csd2d(data, ch_axis=0, delta=1):
+    """estimates traditional current source density (csd) using
```

Comment:
@spbrandt we follow PEP 257: https://peps.python.org/pep-0257/

It should be a single line summary followed by a longer description after leaving one line. Sorry for being so nitpicky ;-)